cmd/infra/powervs/validate.go

```diff
@@ -157,51 +146,52 @@ func validateVpc(vpcName string, resourceGroupID string, v1 *vpcv1.VpcV1) (vpc *
 
 	if !httpOk || !httpsOk {
 		err = fmt.Errorf("vpc security group does not have the required inbound rules, ports 80 and 443 should be allowed")
+		if err != nil {
+			return nil, err
+		}
```

Comment:
here err will always be != nil, hence this can be modified as
```golang
return nil, fmt.Errorf("vpc security group does not have the required inbound rules, ports 80 and 443 should be allowed")
```